nimbus.py

```diff
@@ -7,7 +7,7 @@
 #
 from werkzeug.exceptions import BadRequestKeyError
 from QA import create_qa_mapping, generate_fact_QA
-from nimbus_nlp.NIMBUS_NLP import NIMBUS_NLP
+from nimbus_nlp.NIMBUS_NLP import NimbusNLP
```

Comment:
Yes! That is [python's naming convention for ClassNames](http://google.github.io/styleguide/pyguide.html#316-naming)